docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  taui:
+    build: ./taui
+    volumes:
+      # Mount app source files separately to avoid clobbering node_modules
+      - ./taui/src:/usr/local/src/src
+      - ./taui/configurations:/usr/local/configurations/configurations
+      - ./taui/assets:/usr/local/assets/assets
+      - ./taui/flow-typed:/usr/local/flow-typed/flow-typed
+      - ./taui/package.json:/usr/local/src/package.json
+      - ./taui/yarn.lock:/usr/local/src/yarn.lock
+      - ./taui/config.json:/usr/local/src/config.json
+      - ./taui/__mocks__:/usr/local/src/__mocks__
+      - ./taui/.flowconfig:/usr/local/src/.flowconfig
+      - ./taui/index.html:/usr/local/src/index.html
```

Comment:
I tend to prefer keeping `node_modules` in the container image, rather than installing it locally. This wall of volume mounts is necessary to avoid clobbering the directory. If UA has a preferred way of managing this problem I'd be happy to change it to match your pattern.

Reply:
This looks fragile. I'd rather not have us need to maintain a list of source files. I believe our projects generally sync `node_modules` with the host file system.

Reply:
Perfectly fine. I removed the file-specific mounts and just mounted `taui/src/` instead.